public/app/plugins/panel/geomap/migrations.test.ts

```diff
@@ -55,7 +55,49 @@ describe('Worldmap Migrations', () => {
             "mouseWheelZoom": true,
             "showZoom": true,
           },
-          "layers": [],
+          "layers": [
+            {
+              "config": {
+                "showLegend": true,
+                "style": {
+                  "color": {
+                    "fixed": "dark-green",
+                  },
+                  "opacity": 0.4,
+                  "rotation": {
+                    "fixed": 0,
+                    "max": 360,
+                    "min": -360,
+                    "mode": "mod",
+                  },
+                  "size": {
+                    "fixed": 5,
+                    "max": 30,
+                    "min": 2,
+                  },
+                  "symbol": {
+                    "fixed": "img/icons/marker/circle.svg",
+                    "mode": "fixed",
+                  },
+                  "textConfig": {
+                    "fontSize": 12,
+                    "offsetX": 0,
+                    "offsetY": 0,
+                    "textAlign": "center",
+                    "textBaseline": "middle",
+                  },
+                },
+              },
+              "location": {
+                "gazetteer": "public/gazetteer/countries.json",
+                "lookup": undefined,
+                "mode": "lookup",
+              },
+              "name": "",
+              "tooltip": true,
```

Comment:
Not a high priority, but should this be tied to `stickyLabels: false,`?

Reply:
not tooltip... but potentially setting the text property on markers.  but it is a bit more complicated 😬 so i skipped for now

Reply:
Sounds good to me!